man/marked.1.txt

```diff
@@ -24,65 +24,55 @@ EXAMPLES
 
 OPTIONS
        -o, --output [output]
-              Specify file output. If none is specified, write to stdout.
+	      Specify file output. If none is specified, write to stdout.
 
        -i, --input [input]
-              Specify file input, otherwise use last argument as input file.
-              If no input file is specified, read from stdin.
+	      Specify file input, otherwise use last argument as  input  file.
```

Comment:
More random whitespace. I think there's 5 or 6 other places in this PR so double check.

Reply:
fixed